apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -22,11 +23,19 @@ const AccessManagement = (): JSX.Element => {
 	const columns = getUserColumns();
 	const data: UserRow[] = getUserRows(users);
 
+	const isLoading = isDataLoading(dataStatus);
+
 	return (
 		<PageLayout>
 			<p className={styles["title"]}>Access Management</p>
-			<p className={styles["sub-title"]}>Users</p>
-			<Table<UserRow> columns={columns} data={data} />
+			{isLoading ? (
```

Comment:
Let's move loader inside PageLayout and pass isLoading as parameter

Reply:
Make sense